src/pages/Delivery/index.jsx

```diff
@@ -43,20 +43,12 @@ const Delivery = () => {
         errorPack
         ?
         <Modal
-          isOpen={currentStatus === 'destination_reached' && openDeliveryConfirmedModal}
-          handleClick={() => {
-            toggleDeliveryConfirmedModal(!openDeliveryConfirmedModal)
-            toConfirm()
-          }}
-          actionText="Aceptar"
+          isOpen={true}
+          handleClick={() => { history.push('/check') }}
```

Comment:
@Katsu08 please check if removing the other actions are correct

Reply:
I'm checking

Reply:
> @Katsu08 please check if removing the other actions are correct

They wasn't removed, they still are here: 

https://github.com/Azordev/maps-project/blob/d6abd0af2590873fcaaebe5844fe88f63489d314/src/pages/Delivery/index.jsx#L55-L70

@Israel-Laguan